src/scripts/modules/add_destination.js

```diff
@@ -55,24 +54,22 @@ function updateFormState(fieldName, value) {
   formState[fieldName] = value;
 }
 
-// function storeNew() {
-//   storedCity = localStorage.setItem(
-//     "newDestination",
-//     JSON.stringify(formState)
-//   );
-// }
+function storeNew() {
+  localStorage.setItem("newDestination", JSON.stringify(formState));
```

Comment:
With this, each time we store a new destination, in reality we are replacing the existing one (if it exists). In reality, we want to add it to the already stored. So, instead of storing a single item, we want to store an array of items (destinations). Each time we call `storeNew` we _contact_ it to the current value. 

Please, give it a try to figure it out given these hints.

If not, here's the solution:

<details>

```js
// get list of destinations and parse it
const storedListLS = localStorage.getItem("destinations"); 
const storedList  = JSON.parse(storedListLS)

// update the list with the new destination (formState).
const newList = [...storedListLS, formState]

// Replace the `destinations`, with the new list:
localStorage.setItem("destinations", SON.stringify(newList));
```

</details>

Reply:
I thought ```storeNew()``` can simply store a new item and then ```loadNew()``` new can read it and add it to an array (there was more code that I deleted), but I think I understand now why that doesn't work. What I still don't understand is what I'm supposed to see in the inspector when this works - a new list item for each new destination? A single array?

Reply:
> What I still don't understand is what I'm supposed to see in the inspector when this works

When you get the item, there's nothing to see. When you save the item, you should see the entry replaced with the new destination, as an array

Reply:
Yes I know where to look but what I thought I needed to do is make it so that a new input doesn't replace the previous one, now I'm really confused

Reply:
Each new input is stored in the destinationList so it doesn't need to stick around in LS?